search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -53,16 +53,23 @@
   [ctx path-w-extension params headers body]
   (let [concept-type (concept-type-path-w-extension->concept-type path-w-extension)
         short-scroll-id (get headers (string/lower-case common-routes/SCROLL_ID_HEADER))
-        scroll-id (core-api/get-scroll-id-from-cache ctx short-scroll-id)
+        scroll-id-and-search-params (core-api/get-scroll-id-and-search-params-from-cache ctx short-scroll-id)
+        scroll-id (:scroll-id scroll-id-and-search-params)
+        cached-search-params (:search-params scroll-id-and-search-params)
         ctx (assoc ctx :query-string body :scroll-id scroll-id)
         params (core-api/process-params concept-type params path-w-extension headers mt/xml)
         result-format (:result-format params)
         _ (info (format "Searching for %ss from client %s in format %s with params %s."
                         (name concept-type) (:client-id ctx)
                         (rfh/printable-result-format result-format) (pr-str params)))
         search-params (lp/process-legacy-psa params)
+        search-params (if cached-search-params
+                        (assoc cached-search-params :result-format (:result-format search-params))
```

Comment:
Anything else needs to be kept other than :result-format?   I wrote the ticket https://bugs.earthdata.nasa.gov/browse/CMR-4737 for the baseline, but somehow after the change of this ticket, it worked fine.